src/NHibernate/Mapping/ByCode/Impl/SubclassMapper.cs

```diff
@@ -59,7 +59,7 @@ public void Extends(System.Type baseType)
 			if (!Container.GetBaseTypes().Contains(baseType))
 			{
 				throw new ArgumentOutOfRangeException("baseType",
-				                                      string.Format("{0} is a valid super-class of {1}", baseType, Container));
+													  string.Format("{0} is a valid super-class of {1}", baseType, Container));
```

Comment:
This king of bad reformat could be avoided by:
 - switching to spaces (not going to happen any time soon it seems).
 - putting the first argument on its own new line, which would cause arguments indentation to have only a single additional tab.

Reply:
This is an artifact of applying patch. The original patch had all spaces. I'll fix.

Reply:
Reverted